projects/plugins/jetpack/extensions/shared/external-media/sources/google-photos/index.js

```diff
@@ -50,12 +78,17 @@ function GooglePhotos( props ) {
 		isSessionExpired,
 		createPickerSession,
 		pickerSession,
+		catchAuthErrors,
 	] );
 
 	if ( pickerFeatureEnabled === null || ! isCachedSessionChecked ) {
 		return <MediaLoadingPlaceholder />;
 	}
 
+	if ( authUpgradeRequired ) {
+		return <GooglePhotosAuthUpgrade { ...props } />;
+	}
```

Comment:
Maybe this condition is better to place after the isAuthenticated check block?

Reply:
Agree! I have reordered it.